pkg/transform/kubernetes/kubernetes.go

```diff
@@ -289,6 +289,53 @@ func populateContainerHealth(app *spec.App) error {
 	return nil
 }
 
+func populateEnvFrom(app *spec.App) error {
+	// iterate on the containers so that we can extract envFrom
+	// that we have custom defined
+	for ci, c := range app.Containers {
+		for ei, e := range c.EnvFrom {
+			cmName := e.ConfigMapRef.Name
+
+			// we also need to check if the configMap specified if it exists
+			var cmFound bool
+			// to populate the envs we also need to know the data
+			// from configmaps defined in the app
+			for _, cm := range app.ConfigMaps {
+				// we will only populate the configMap that is specified
+				// not every configMap out there
+				if cm.Name != cmName {
+					continue
+				}
+				// start populating
+				for k, _ := range cm.Data {
+					// here we are directly referring to the containers
+					// from app.PodSpec.Containers because that is where data
+					// is parsed into so populating that is more valid thing to do
+					app.PodSpec.Containers[ci].Env = append(app.PodSpec.Containers[ci].Env, api_v1.EnvVar{
```

Comment:
I'm a bit confused here, why are we not populating this in app.Container? That is going in the PodSpec in `createDeployment()` anyway, so what's the harm in not putting this in app.Container?